library/ssl_tls13_server.c

```diff
@@ -2801,19 +2799,15 @@ static int ssl_tls13_write_certificate_verify(mbedtls_ssl_context *ssl)
 static void ssl_tls13_prepare_for_handshake_second_flight(
     mbedtls_ssl_context *ssl)
 {
-    MBEDTLS_SSL_DEBUG_MSG(
-        2, ("=> ssl_tls13_prepare_for_handshake_second_flight"));
-
     if (ssl->handshake->certificate_request_sent) {
         mbedtls_ssl_handshake_set_state(ssl, MBEDTLS_SSL_CLIENT_CERTIFICATE);
     } else {
-        MBEDTLS_SSL_DEBUG_MSG(2, ("skip parse certificate"));
         MBEDTLS_SSL_DEBUG_MSG(2, ("skip parse certificate verify"));
```

Comment:
```suggestion
```
included in the following message now

Reply:
Sorry, I've just seen that "skip parse certificate" cannot be removed as used in tests. Then just keep what was there before my comment I'd say:
```
        MBEDTLS_SSL_DEBUG_MSG(2, ("skip parse certificate"));
        MBEDTLS_SSL_DEBUG_MSG(2, ("skip parse certificate verify"));
```
Sorry for the noise.